components/03-organisms/menu/utility-nav/yds-utility-nav.twig

```diff
@@ -2,21 +2,22 @@
 
 <div {{ bem('utility-bar') }}>
   {# Menu #}
-  {% include "@molecules/menu/yds-menu.twig" with {
-    menu__base_class: utility_nav__base_class,
-    menu__name: 'Utility',
-    items: utility_nav__items|default(items),
-  } %}
+  {% block utility_nav__menu %}
+    {% include "@organisms/menu/utility-nav/_utility-nav--menu.twig" %}
+  {% endblock %}
   {# Search #}
+  {% set utility_nav__search = true %}
```

Comment:
We can't do this here, or it will force search to be on for every site. The `if` below is there so that sites can choose whether or not they want to have a site search.

Many of the sites on the platform will be essentially "brochureware sites" with only a handful of pages, and search would seem ridiculous. So we want to provide a toggle in the Drupal Settings page for them to turn search on or off.

Reply:
Completely understand - this is removed in YALB-702 and turned into a dynamic variable that comes from the theme settings form. This was hardcoded here so that we could test YALB-700.